src/aioswitcher/device/__init__.py

```diff
@@ -354,3 +373,15 @@ def __post_init__(self) -> None:
         if self.device_type.category != DeviceCategory.SHUTTER:
             raise ValueError("only shutters are allowed")
         return super().__post_init__()
+
+
+@final
+@dataclass
+class SwitcherS12(SwitcherS12Base, SwitcherBase):
+    """Implementation of the Switcher Shutter S12 device."""
+
+    def __post_init__(self) -> None:
+        """Post initialization validate device type category as SHUTTER."""
+        if self.device_type.category != DeviceCategory.S12:
```

Comment:
This is not a category but a device model, does not align with our current design